### DISCUSSION TITLE
UnCommon Packages

### DISCUSSION THREAD

**Discussion by danielsherratt** [microsoft/winget-pkgs#87961] [General] [answered]
Hi All,

I'm moving away from a group policy based install script to install some of our packages
A lot of them are quite old and no longer really exist online for redownloading

What's your thoughts on spinning up a VPS and hosting the exe's on it, so www.myvps.nz/repo/myoldpackage.exe
These packages don't exist in winget so I suspect the usage would be low

Would it be worth prefixing the id with something unique? eg dan-myoldpackage

Thanks,
Daniel

**Comment by mdanish-kh** [comment, marked as answer]
As per https://github.com/microsoft/winget-pkgs/pull/16368#issuecomment-857281310 , the download URLs need to come directly from the ISV's (Independent Software Vendor) Server, and if the official download URLs aren't available, the packages won't be accepted in winget.

**Reply by danielsherratt** [reply]
OK - thanks, Perhaps if they don't exist at a reputable source, do they really need them, they are quite old
